Added check for /var/run/vpn directory.

jaredcrean opened the pull request:
everytime I rebooted the /var/run/directory would disappear. So I added a check for that.

Commit:
Update pia-launch.sh

Commit:
Update pia-launch.sh

Commit:
Added a check for the /var/run/vpn directory

Added a check for the /var/run/vpn directory for the pid file.

dh4 commented:
What configuration are you running that needs this directory? It shouldn't be needed by anything within the script.

/var/run being cleared on reboot is by design:
http://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s13.html
http://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html

jaredcrean commented:
Dan,

I had some local configurations, I had changed a while back that I forgot about. I removed them and it run's fine that is my bad. I will cancel the pull request. Thanks for making this script by the way.

dh4 commented:
No problem. Glad you find it useful.
